src/components/DayPickerRangeController.jsx

```diff
@@ -52,6 +52,7 @@ const propTypes = forbidExtraProps({
   onOutsideClick: PropTypes.func,
   renderDay: PropTypes.func,
   renderCalendarInfo: PropTypes.func,
+  firstDayOfWeek: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6]),
```

Comment:
I think this propType should be shared and exported up from the components that need it - ie, this component shouldn't du-locate the definition, it should just import the propType from DayPicker and delegate the authority to it.

Same applies to any component where the prop is merely proxied down.

Effectively I'd prefer the oneOf to only appear in one place.